content/blog/2024/01/2024-01-30-2024-roadmap-and-fundraiser.md

```diff
@@ -0,0 +1,78 @@
+
+Element used to subsidize our activity, and still donates upwards of £3M in core development annually, but we are paying our own bills now. While we managed a 400+% increase in revenue to £364K last year, at £440K our expenses exceeded our revenue. As we enter our first full year of paying our own bills, the full cost of operating the Foundation at its current scope is projected to stabilize somewhere north of £900K in 2024. And the more we can raise beyond £900K, the less dependent the ecosystem is on Element.
+
+As I get more settled in, we'll deliver more granular financial projections and reporting. For now, the rough breakdown of those projected expenses:
+
+- £550K in staffing 4 full-time people covering management, advocacy, standards development, and trust and safety, as well as external part-time staff for core project and bridge maintenance and infra ops
+- £250K in servers and other third-party providers
+- £100K in administrative costs covering legal, accounting, and compliance
+
+To put that £900K figure in perspective, the [Python Software Foundation's expenses](https://www.python.org/psf/records/) are nearer £2.9M, and [Signal's annual costs](https://signal.org/blog/signal-is-expensive/) are in the ballpark of £39M!
+
+**This year is exciting and full of promise, and there are many reasons to be hopeful. If the work we are doing to support Matrix is valuable to you, we need your help to continue it well past 2024. We need more organizations to [step up as members](https://matrix.org/membership/) to invest in the Matrix ecosystem. Otherwise, we need to start scaling back programs and staff starting January 2025.**
+
+We're going to publish a series of blog posts over the next three months to unpack our work and our roadmap for the future. You'll see posts covering: governance, ecosystem diversity and community engagement, infrastructure, trust and safety, fighting for your digital rights, and more.
+
+With your support, the Foundation can charge full steam ahead to deliver on the promise of making open, unencumbered, safe, secure, and private communications technology.
+
+## A hint of the year ahead
+
+The forthcoming posts in this series will dive into several of our core program areas, and explore where we aim to go. But when I think about the year ahead, three overriding themes stand out to me:
+
+- Ramp up open governance of Matrix and the Foundation
+- Prioritize Trust & Safety
+- Cement the Foundation as an independent and sustainable organization
+
+2024 will see the Foundation sustain and deepen work in our ongoing responsibilities:
+
+- Empowering the [Spec Core Team (SCT)](https://matrix.org/about/#the-spec-core-team) to deliver an effective, modern spec change process
+- Running the Matrix.org homeserver and public bridges
+- Trust & Safety
+- Communications and community engagement
+- Compliance
+- Governance
+- Fundraising
```

Comment:
This whole section feels somewhat like a repetition is that intentional?

Reply:
Yeah intentional, making it so that folks who only skim will still find what they're looking for.